stretch_core/rviz/stretch_simple_test.rviz

```diff
@@ -1,402 +1,103 @@
 Panels:
```

Comment:
No need to read this line for line, but this changes the `rviz` namespace to either `rviz_common/rviz_default_plugins` and removes the default value for many of the fields.